spec/models/response_map_spec.rb

```
@@ -1,6 +1,147 @@
+    before do
+      reviewee_participant.update_column(:updated_at, base_time)
+      team.update_column(:updated_at, base_time)
+      teams_participant_record.update_column(:updated_at, base_time)
+    end
+
+    it 'returns true when no submitted response exists yet' do
+      expect(response_map.has_submission_been_updated?).to be true
+    end
+
+    it 'returns false when the last submitted response is the most recent activity' do
+      Response.create!(map_id: response_map.id, is_submitted: true, created_at: response_time, updated_at: response_time)
```

RuboCop (1.86.1)

[convention] Line is too long. [121/120]

(Layout/LineLength)